### ISSUE TITLE
Feature Request: Check if files have been deleted

### ISSUE THREAD

**Issue by ckab74** [oxygenxml/oxygen-dita-translation-package-builder#26]
A request from our localization department.
It would be great if the Translation Package Builder could check if files have been deleted since the last milestone was created.

**Comment by raducoravu** [comment]
Christina, thanks for adding the issue, can you tell me the current behavior of the builder when the file gets deleted? And what you want the behavior to be.

**Comment by ckab74** [comment]
Hi Radu,
Current behavior: when a file gets deleted, the ditamap in which the file was originally linked is changed accordingly. The TPB treats this file correctly and marks it as changed. But, for example, an author wouldn't know which file that is when the file has been deleted several weeks ago. So an author would have to diff the original ditamap and the latest one to see which files are no longer available - an extra step that would have to be done for all languages. If no diff is done, the deleted files stay in the language folders forever.
I could imagine that at least the report that is generated during the "Create modified files package" step could show a list of files that have been removed from a ditamap. Not sure if this is feasible, but if the authors could get a list of deleted files, they would have a better overview of what needs to be changed in the folders with translated files. 
Not sure if the Apply Packages function could handle the deleted files when the translated files are copied to the translation folders.
I hope I've made myself clear enough.

**Comment by raducoravu** [comment]
Yes, it's clear, we'll try to improve the generated HTML report.
One more thing, if you right click a DITA Map root element in the DITA Maps manager view there is a "Find Unreferenced Resources" action which can be run on the translated DITA Maps to find the topics and other resources (images for example) which are not used.

**Comment by ckab74** [comment]
Thanks, that would be great. 
Yes, we know that function and think it's very helpful.
